packages/dashboard-frontend/src/services/oauth/index.ts

```diff
@@ -62,8 +62,8 @@ export default class OAuthService {
           response.data.attributes.oauth_authentication_url,
           redirectUrl.toString(),
         );
+        throw e;
       }
-      throw e;
```

Comment:
@vinokurig would it be possible to add test for this functionality? Do we need to re-throw exception in general?

Reply:
> would it be possible to add test for this functionality?

done
>Do we need to re-throw exception in general?

Yes, the exception interrupts the workspace start event until the authorisation callback.